db/migrate/20201214073537_create_bug_reports.rb

```diff
@@ -0,0 +1,10 @@
+class CreateBugReports < ActiveRecord::Migration[6.1]
+  def change
+    create_table :bug_reports do |t|
+      t.belongs_to :user, null: false
+      t.text :body, null: false
```

Comment:
Rename to `content` pls. Also, let's make this markdown, so can you rename to `content_markdown` and add a field for `content_html` which gets set in the `before_update` block pls?